src/SFA.DAS.EmployerIncentives.Api/HealthChecks/InnerApiHealthCheck.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace SFA.DAS.EmployerIncentives.Api.HealthChecks
+{
+    public class InnerApiHealthCheck : IHealthCheck
```

Comment:
Doesn't seem very useful this healthcheck

Reply:
This was a stub, it's now got an implementation

Reply:
fixed